client/my-sites/email/email-non-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { SiteDetails } from '@automattic/data-stores';
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import { stringify } from 'qs';
+import { useDomainOwnerUserName } from 'calypso/components/data/query-domain-owner-username';
+import PromoCard from 'calypso/components/promo-section/promo-card';
+import TrackComponentView from 'calypso/lib/analytics/track-component-view';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ResponseDomain } from 'calypso/lib/domains/types';
+import { emailManagementPurchaseNewEmailAccount } from 'calypso/my-sites/email/paths';
+
+import './style.scss';
+
+type EmailNonDomainOwnerMessageProps = {
+	domain?: ResponseDomain;
+	selectedSite?: SiteDetails | null;
+};
+
+const buildQueryString = ( parameters = {} ) =>
+	parameters ? stringify( parameters, { addQueryPrefix: true, skipNulls: true } ) : '';
+
+export const EmailNonDomainOwnerMessage = ( props: EmailNonDomainOwnerMessageProps ) => {
+	const { domain, selectedSite } = props;
+
+	const translate = useTranslate();
+
+	const ownerUserName = useDomainOwnerUserName( selectedSite, domain );
+
+	const isPrivacyAvailable = domain?.privacyAvailable;
+
+	const buildLoginUrl = () => {
+		const redirectUrlParameter = emailManagementPurchaseNewEmailAccount(
+			selectedSite?.slug ?? '',
+			domain?.name ?? '',
+			null,
+			'login-redirect'
+		);
+
+		return `/log-in/${ buildQueryString( {
+			email_address: ownerUserName,
+			redirect_to: redirectUrlParameter,
+		} ) }`;
+	};
+
+	const contactOwnerUrl = `https://privatewho.is/${ buildQueryString( {
+		s: domain?.name,
+	} ) }`;
+
+	const loginUrl = buildLoginUrl();
+
+	const onClickLink = ( eventType: 'contact' | 'login' | 'support' ) => {
+		const properties = {
+			action: eventType,
```

Comment:
```suggestion
			action: eventType,
			context: 'non-domain-owner'
```

Let's include the context here, so we can differentiate the clicks and impressions from non-domain-owner and non-email-owner.